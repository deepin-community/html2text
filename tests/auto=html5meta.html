<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US">
<head>
<meta charset="utf-8" />
	<title>Recognition of HTML5 META charset tag</title>
</head>
<body>
<!--  See, e.g.:
	https://www.w3.org/International/questions/qa-html-encoding-declarations
-->
<p>
Should be Chinese: <span lang="zh">你好!</span>
</p>
<p>
Should be Russian: <span lang="ru">Привет!</span>
</p>
<p>
Should be Greek: <span lang="el">Γεια!</span>
</p>
<p>
Degrees centigrade: ℃, <br />
Ohms sign: Ω, <br />
Angstroms sign: Å
</p>
</body>
</html>

