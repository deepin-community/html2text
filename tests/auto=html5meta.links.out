auto=html5meta.default.out